pygenometracks/plotTracks.py

```diff
@@ -296,8 +304,10 @@ def main(args=None):
             sys.stderr.write("saving {}\n".format(file_name))
             # print("{} {} {}".format(chrom, start, end))
             trp.plot(file_name, chrom, start, end, title=args.title,
-                     h_align_titles=args.trackLabelHAlign)
+                     h_align_titles=args.trackLabelHAlign,
+                     decreasingXAxis=args.decreasingXAxis)
```

Comment:
the variable should be `decreasing_x_axis`, maybe?

Reply:
You are right.